packages/mui-codemod/src/v6.0.0/sx-prop/sx-v6.test.js

```diff
@@ -0,0 +1,109 @@
+import path from 'path';
+import { expect } from 'chai';
+import { jscodeshift } from '../../../testUtils';
+import transform from './sx-v6';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('v6.0.0', () => {
+    describe('basic sx-v6', () => {
```

Comment:
Could use: https://github.com/mui/material-ui/blob/next/packages/mui-codemod/testUtils/index.js#L13

Reply:
I did not know it existed. To be honest, I prefer writing test separately instead of using `.forEach`. The reason is debugging.

With multiple `it()`, you can use `.skip` or `.only` to debug a test, but it's not possible with `.forEach`.